dockstore-webservice/src/main/java/io/dockstore/webservice/api/UpdateAITopicRequest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2024 OICR and UCSC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *           http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.dockstore.webservice.api;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.media.Schema.RequiredMode;
+import jakarta.validation.constraints.NotNull;
+
+@Schema(description = "The AI topic update request")
+public class UpdateAITopicRequest {
+
+    @NotNull
+    @Schema(description = "The AI topic", requiredMode = RequiredMode.REQUIRED)
```

Comment:
I went with the silent truncation to match the behaviour of `topicAutomatic`, which silently truncates topics greater than `TOPIC_LENGTH`. I'll add a description saying that topics > `TOPIC_LENGTH` are truncated.

> Also, WorkflowResource.updateWorkflow(), which takes a workflow as a parameter, will I believe enforce a TOPIC_LIMIT by not deserializing a workflow with a greater than TOPIC_LENGTH ai topic.

That's true if `maxLength` is set, so perhaps it's another reason why we shouldn't set it since the AI topic can't be updated with `WorkflowResource.updateWorkflow()` anymore